Makefile

```diff
@@ -160,8 +160,10 @@ GOTESTSUM_ARGS += --junitfile=$(ARTIFACT_DIR)/junit.xml
 endif
 
 GO_TEST = go test
+GO_TEST_SUB = go test
 ifdef USE_GOTESTSUM
 GO_TEST = $(GOTESTSUM) $(GOTESTSUM_ARGS) --
+GO_TEST_SUB = ../../${GO_TEST}
```

Comment:
this is hacky. Just reference the toplevel dir somehow, or make TOOLS_DIR absolute

Reply:
updated